pwndbg/file.py

```diff
@@ -33,7 +33,7 @@ def get_file(path, recurse=1):
     local_path = path
 
     if pwndbg.qemu.root() and recurse:
-        return get(os.path.join(pwndbg.qemu.binfmt_root, path), 0)
+        return os.path.join(pwndbg.qemu.binfmt_root, path)
```

Comment:
So it turns out that instead of returning local path we could have returned bytes read from the file.

Uh...

Reply:
Btw shouldn't have the the if order reversed, i.e. first check for remote and then for `pwndbg.qemu.root()`?

Also, what is the situation when `pwndbg.qemu.root()` is used? I have qemu user on my system but I don't have the `/etc/qemu-binfmt` it tries to check :S.

Reply:
It's arbitrarily set by debian version of qemu as the default value of `QEMU_LD_PREFIX`. `qemu-arm-static --help` is showing it.

```
~/qemu-2.11+dfsg $ grep -r interp-prefix
debian/rules:   --interp-prefix=/etc/qemu-binfmt/%M \
...
```